scripts/policy/protocols/conn/known-services.zeek

```diff
@@ -298,6 +298,11 @@ event protocol_confirmation(c: connection, atype: Analyzer::Tag, aid: count) &pr
 	known_services_done(c);
 	}
 
+event analyzer_confirmation(c: connection, atype: AllAnalyzers::Tag, aid: count) &priority=-5
```

Comment:
Yes, because if someone elsewhere is still emitting the `protocol_confirmation` events in a plugin, removing them would break things, right?

Reply:
I could add something somewhere that basically just reschedules any `protocol_confirmation`/`protocol_violation` events as the `analyzer` versions, and remove all of these. It'd have a slight performance difference since the event wouldn't get handled until the next cycle, but that might be ok?

Reply:
I removed the handling of the `protocol_` events from all of the base scripts, and changed the two methods in `Analyzer` to just enqueue both the old and new events.